sys/docker/Dockerfile

```diff
@@ -42,42 +41,44 @@ RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - \
     && npm rebuild node-sass \
     && rm -rf /var/lib/apt/lists/*
 
-RUN apt-get -y remove \
-        git \
-        gcc \
-        g++ \
-        make \
-    && apt-get -y clean \
-    && apt-get -y autoremove \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY sys/docker/start.sh ./
-
 RUN apt-get update \
     && apt-get install -y --no-install-recommends nginx \
     && mkdir -p /data/nginx/cache \
     && rm -rf /var/lib/apt/lists/*
 
-COPY ./sys/nginx/nginx.conf /etc/nginx/conf.d/default.conf
-
 RUN echo "* * * * *", command: "find /data/nginx/cache/ -type f -newermt '30 seconds ago' | xargs rm" >> /var/spool/cron/crontabs/root \
```

Comment:
`find /data/nginx/cache/ -type f -newermt '30 seconds ago' -exec rm -f {} \;` should be better

`find /data/nginx/cache/ -type f -newermt '30 seconds ago' -delete` should be the best one

`find /data/nginx/cache/ -type f -newermt '30 seconds ago' -print0 | xargs -0 rm -f` should be a more robust implementation of the current approach